README.md

```diff
@@ -109,16 +113,19 @@ The Flask App offers 3 endpoints that returns some text `FLASK_SERVICE_IP:5000/`
 
 Run ```kubectl get services``` to find the [FLASK_SERVICE_IP](https://cl.ly/a344b20d5481) address of the flask application service
 
+(v minor edit @z: could combine these two sections to just run the ```kubectl get services``` command once, and grab all the output, since it gives all IPs for this "Use the Flask App" and "Use the Springboot App")
+
 # Use the SpringBoot App
 
-The SpringBoot App offers 3 endpoints that returns some text `SpringBoot_SERVICE_IP:8080/`, `SpringBoot_SERVICE_IP:8080/query`
+The SpringBoot App offers 2 endpoints that returns some text `SpringBoot_SERVICE_IP:8080/`, `SpringBoot_SERVICE_IP:8080/query`
 
 Run ```kubectl get services``` to find the [SpringBoot_SERVICE_IP](https://cl.ly/a344b20d5481) address of the flask application service
 
 ## Cluster Accessible via Internet
 If you used the default GKE template or know the cluster is accessible via the internet, you can use the IP found in the `EXTERNAL-IP` column as SERVICE_IP
 
-then hit one of the following:
+then hit one of the following: (@z: this didn't work for me - however, curl `Flask_SERVICE_IP:5000/` and `Flask_SERVICE_IP:5000/query` yielded 'Flask has been kuberneted' and '(u'1', u'z@datadoghq.com')', respectively - what's the intended outcome here?)
```

Comment:
Good point to add for clarity (expected outcome - unsure why springboot didn't work for you though)